tests/features/content_lock.feature

```diff
@@ -0,0 +1,16 @@
+@api @javascript
+Feature: Content lock
+  as a content editor
+  I should be prevented from concurrently editing a node
+  so that no work is lost
+
+  @content_lock
+  Scenario: Content gets lock and lock can be broken
+    Given I am logged in as a user with the "Author" role
+    And I am viewing my "oe_workflow_demo" with the title "Demo site"
```

Comment:
This step should be more business readable, we could also create a custom one if needed.

Reply:
My problem is mostly with `oe_workflow_demo`, as in: no business person will know what we refer to. Maybe something like this?

```gherkin
And I am viewing a content page with title "Demo site"
```